site/tests/fake_datacite_client.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
```

Comment:
Once again I had to copy the whole mocked client from invenio-rdm-records. It's under `invenio_rdm_records/tests` and therefore is not resolvable for importing.